.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: android
+sudo: true # travis-ci/travis-ci#5582 implies this could help against killed build-jobs
 before_script: ./pre-build.sh && sed -i "s#com.android.tools.build:gradle:.*'#com.android.tools.build:gradle:1.5.0'#" build.gradle
-jdk: openjdk7
+jdk:
+    - openjdk7
+    - oraclejdk7
+    - oraclejdk8
```

Comment:
You are right. I thought of maybe checking compatibility with different build setups, but since this triples the tasks execution time in the worst case I will remove this.